src/e2e-node/e2e.test.ts

```diff
@@ -352,6 +358,8 @@ const serversUnderTest: AuthDetails[] = [
 describe.each(serversUnderTest)(
   "Authenticated end-to-end tests against Pod [%s] and OIDC Issuer [%s]:",
   (rootContainer, oidcIssuer, clientId, clientSecret, refreshToken) => {
+    rootContainer = "https://" + rootContainer;
```

Comment:
Maybe just a matching comment to explain why we have to stitch back in the `https://` parts here...?

Reply:
Good one: 854876a8